packages/widget/src/pages/TransactionDetailsPage/SupportIdCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { ContentCopyRounded, OpenInNew } from '@mui/icons-material';
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Card } from '../../components/Card/Card.js';
+import { CardIconButton } from '../../components/Card/CardIconButton.js';
+import { CardTitle } from '../../components/Card/CardTitle.js';
+import { useWidgetConfig } from '../../providers/WidgetProvider/WidgetProvider.js';
+
+interface SupportIdCardProps {
+  supportId: string;
+}
+
+const getTxHash = (supportId: string) =>
+  supportId.indexOf('_') !== -1
+    ? supportId.substring(0, supportId.indexOf('_'))
+    : supportId;
+
+export const SupportIdCard = ({ supportId }: SupportIdCardProps) => {
+  const { t } = useTranslation();
+
+  const { explorerUrl } = useWidgetConfig();
+
+  const copySupportId = async () => {
+    await navigator.clipboard.writeText(supportId);
+  };
+
+  const openSupportIdInExplorer = () => {
+    const txHash = getTxHash(supportId);
+    window.open(`${explorerUrl}${txHash}`, '_blank');
+  };
+
+  return (
+    <Card sx={{ marginTop: 2 }}>
+      <Box
+        sx={{
+          display: 'flex',
+          flex: 1,
+        }}
+      >
+        <CardTitle flex={1}>{t('main.supportId')}</CardTitle>
+        <Box
+          sx={{
+            gap: 1,
+            display: 'flex',
+            marginRight: 2,
+            marginTop: 1,
+          }}
+        >
+          <CardIconButton size="small" onClick={copySupportId}>
+            <ContentCopyRounded fontSize="inherit" />
+          </CardIconButton>
+          {explorerUrl ? (
```

Comment:
Currently the button for the link will only appear if an `exploreUrl` is featured in the config - there is a question here as to whether we would want `explorerUrl` to default to `https://scan.li.fi/tx/` if explorerUrl is not added to the Widget config?

Reply:
Yes please, let's always fallback to li.fi scan